src/AST/ASTNode.ts

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) ZALORA
+//
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+interface ASTNode {
+  tokenLiteral(): string;
+  description?: string;
```

Comment:
Just use Object.toString(), then we don't need to specify it here. Let's remove.